integr8ly/application-monitoring-operator
# Bump operator-sdk version to 0.15.2 and AMO to 1.2.0 for release

Needed for a new release to prevent pod duplication as described in https://issues.redhat.com/browse/INTLY-8046
- Upgraded following the [operator-sdk upgrade guide](https://sdk.operatorframework.io/docs/migration/version-upgrade-guide/#v015x)
## Changes
- Update operator-sdk to version 0.15.2
- Small changes in `make gen/csv` recipe to use `operator-sdk generate csv` instead of `olm-catalog` which has been deprecated.
- Manifests are now located in `deploy/olm-catalog/application-monitoring-operator` directory, and previous versions are no longer stored in-project.
- Deleting older manifests (due to the new way the operator-sdk outputs resources).

## Verification
To ensure the newer version of the operator-sdk works as expected 
- run `make setup/travis`
- run `make code/gen` 
- run `make code/run` 
- run `make image/build`
- run `make image/build/test`
